env.go

```diff
@@ -0,0 +1,41 @@
+package envconf
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"io"
+	"os"
+)
+
+var (
+	ErrInvalidPair = errors.New("invalid pair for env variable")
+)
+
+type EnvConfig struct{}
+
+func NewEnvConf() *EnvConfig {
+	return &EnvConfig{}
+}
+
+func (e *EnvConfig) SetEnv(data io.Reader) error {
+	r := bufio.NewReader(data)
+	for {
+		pair, err := r.ReadBytes('\n')
+		if err != nil && err != io.EOF {
+			return err
+		}
+		if err == io.EOF && len(pair) == 0 {
+			return nil
```

Comment:
if err != nil {
    if err == io.EOF && len(pair) == 0 {
			return nil
    }
    return err
}

Reply:
Here you made error (proebalis`)